python/instrumentation/openinference-instrumentation-beeai/poetry.lock

```diff
@@ -0,0 +1,2339 @@
+# This file is automatically @generated by Poetry 2.1.1 and should not be changed by hand.
```

Comment:
Is the poetry lock file desired here? Just making sure.

Reply:
It is not needed. Removed.